src/platform/platform.base.js

```diff
@@ -51,14 +51,15 @@ export default class BasePlatform {
 
   /**
 	 * Returns the maximum size in pixels of given canvas element.
-	 * @param {HTMLCanvasElement} element
+	 * @param {?HTMLCanvasElement} element
```

Comment:
Looking at the jsdoc docs, this does not seem correct

 ![image](https://github.com/chartjs/Chart.js/assets/39033624/526db988-b48b-454f-bf28-3a7378ef1d24)

Reply:
I fixed this in ea5403d (I wish everything was consistently typescript to avoid multiple standard on documentation)